common/tools/dev-tool/src/commands/run/testBrowser.ts

```diff
@@ -10,7 +11,9 @@ export const commandInfo = makeCommandInfo(
 );
 
 export default leafCommand(commandInfo, async (options) => {
-  const karmaArgs = options["--"]?.length ? options["--"]?.join(" ") : "--single-run";
+  const karmaArgs = options["--"]?.length
+    ? options["--"]?.join(" ")
+    : `${(await isModuleProject()) ? "karma.conf.cjs" : ""} --single-run`;
```

Comment:
Ah I see you already embedded this into the command, then please ignore that suggestion in package.json.